### ISSUE TITLE
rule, sql escape variables, not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#781]
should report as errors,

```abap
    UPDATE zabaplint_pack
      SET json = iv_json
      WHERE devclass = iv_devclass.
```

```abap
DELETE FROM zabaplint_pack WHERE devclass = iv_devclass.
```

**Comment by larshp** [comment]
2.57.0
